packages/filecoin-api/src/aggregator/api.ts

```diff
@@ -0,0 +1,325 @@
+   * `bagy...aggregate` Piece CID of an aggregate
+   */
+  aggregate: PieceLink
+  /**
+   * `bafy...cbor` as CID of dag-cbor block with list of pieces in an aggregate.
+   */
+  pieces: Link
+  /**
+   * Grouping information for submitted piece.
+   */
+  group: string
+  /**
+   * Insertion date in milliseconds since unix epoch.
+   */
+  insertedAt: number
+}
+
+// TODO: probably group should also be key!
+export interface AggregateRecordKey
+  extends Pick<AggregateRecord, 'aggregate'> {}
+
+export interface InclusionRecord {
+  /**
+   * Piece CID for the content.
+   */
+  piece: PieceLink
+  /**
+   * Piece CID of an aggregate.
+   */
+  aggregate: PieceLink
+  /**
+   * Grouping information for submitted piece.
+   */
+  group: string
+  /**
+   * Proof that the piece is included in the aggregate.
+   */
+  inclusion: InclusionProof
+  /**
+   * Insertion date in milliseconds since unix epoch.
+   */
+  insertedAt: number
+}
+
+export interface InclusionRecordKey
+  extends Pick<InclusionRecord, 'piece' | 'aggregate'> {}
+
+export interface InclusionRecordQueryByGroup
+  extends Pick<InclusionRecord, 'piece' | 'group'> {}
+
+export type BufferedPiece = {
+  /**
+   * Piece CID for the content.
+   */
+  piece: PieceLink
+  /**
+   * Policies that this piece is under
+   */
+  policy: PiecePolicy
+  /**
+   * Insertion date in milliseconds since unix epoch.
+   */
+  insertedAt: number
+}
+
+export interface Buffer {
+  /**
+   * `bagy...aggregate` Piece CID of an aggregate
+   */
+  aggregate?: PieceLink
+  /**
+   * Pieces inside the buffer record.
+   */
+  pieces: BufferedPiece[]
+  /**
+   * Grouping information for submitted buffer.
+   */
+  group: string
+}
+
+export interface BufferRecord {
+  /**
+   * Buffer with a set of Filecoin pieces pending aggregation.
+   */
+  buffer: Buffer
+  /**
+   * `bafy...cbor` as CID of dag-cbor block with list of pieces in an aggregate.
+   */
+  block: Link
+}
+
+export interface BufferMessage {
+  /**
+   * `bagy...aggregate` Piece CID of an aggregate
+   */
+  aggregate?: PieceLink
+  /**
+   * `bafy...cbor` as CID of dag-cbor block with Buffer
+   */
+  pieces: Link
+  /**
+   * Grouping information for submitted buffer.
+   */
+  group: string
+}
+
+export interface PieceAcceptMessage {
+  /**
+   * Piece CID.
+   */
+  piece: PieceLink
+  /**
+   * Piece CID of an aggregate
+   */
+  aggregate: PieceLink
+  /**
+   * Grouping information for submitted piece.
+   */
+  group: string
+  /**
+   * Proof that the piece is included in the aggregate.
+   */
+  inclusion: InclusionProof
+}
+
+export interface AggregateOfferMessage {
+  /**
+   * Link to the task invocation for `aggregate/offer`.
+   */
+  // TODO: do we need?
+  task?: Link
+  /**
+   * Piece CID of an aggregate.
+   */
+  aggregate: PieceLink
+  /**
+   * List of pieces in an aggregate.
+   */
+  pieces: Link
+  /**
+   * Grouping information for submitted piece.
+   */
+  group: string
+}
+
+export interface TestEventsContext
```

Comment:
I would prefer not to given it is specific per service.

Otherwise, need to important all the interfaces into the test and same names for same messages like `PieceMessageContext` make it more weird.

Happy to reconsider if you feel strongly :)